### ISSUE TITLE
master isn't building via maven

### ISSUE THREAD

**Issue by nitinverma** [cbeust/testng#947]
```
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building TestNG 6.9.11-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for org.apache.maven.plugins:maven-jar-plugin:jar:3.0.0-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.747 s
[INFO] Finished at: 2016-01-04T12:10:08+05:30
[INFO] Final Memory: 8M/156M
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.apache.maven.plugins:maven-jar-plugin:3.0.0-SNAPSHOT or one of its dependencies could not be resolved: Could not find artifact org.apache.maven.plugins:maven-jar-plugin:jar:3.0.0-SNAPSHOT -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

**Comment by nitinverma** [comment]
`org.apache.maven.plugins:maven-jar-plugin:jar:3.0.0-SNAPSHOT` isn't publicly available.

**Comment by cbeust** [comment]
It's building with gradle and kobalt, I haven't used mvn to build TestNG in a while...

**Comment by nitinverma** [comment]
As https://github.com/nhojpatrick/testng/commit/570aeb9d1cbb5089122e2a4407b18660323d307e only changes `pom.xml` breaks maven build.

**Comment by nitinverma** [comment]
just a thought
1. if we have three builds then all should run symmetric (same dependences and options)
2. else we should deprecate and delete what we deem legacy.
just gives more clarity to the forker.

**Comment by cbeust** [comment]
Yes, I should probably delete pom.xml since it’s really not used at all any
more.
​
-- 
Cédric

**Comment by nitinverma** [comment]
:+1:

**Comment by nitinverma** [comment]
off-topic: can kobalt be used for android builds? flavors/build-types/progaurd/dependency{jar,aar}

**Comment by juherr** [comment]
> I should probably delete pom.xml since it’s really not used at all any more.

I cant' remember why, but @missedone asked it some times ago.

**Comment by nhojpatrick** [comment]
The pull request was merged in too soon.

I submitted a pull request for maven-jar-plugin, if that is accepted and goes into the current release then that will be 3.0.0-SNAPSHOT. If testng only accepts non SNAPSHOT versions then it should also wait till 3.0.0 is released and I would update the pull request with those changes.

**Comment by cbeust** [comment]
@nitinverma Yes, Kobalt builds android apps. See for example u2020 (which I forked to build with kobalt):

https://github.com/cbeust/u2020/blob/build-with-kobalt/kobalt/src/Build.kt

The plug-in is now in its own project:

https://github.com/cbeust/kobalt-android

Note: there's a regression right now and u2020 doesn't see to run once deployed on the device, will look into it soon, but the bottom line is that pretty much everything is supported by kobalt-android:

- resource and manifest merging
- predex and dex
- product flavors and build types
- retrolambda
- (proguard not yet)

**Comment by nitinverma** [comment]
Nice, Thanx... Build.kt seems easy to read. Will have to see how to use fabric.io plugin with kobalt.

**Comment by missedone** [comment]
i was used to import testng into eclipse as maven project, and build it with maven.
i am ok to remove pom.xml (and maven build script) since we can off load the support/maintain effort on maven build.
thanks

**Comment by nitinverma** [comment]
We can close this. Maven build is back in business after the revert.

**Comment by juherr** [comment]
Fixed by #948
